src/map/skill.cpp

```diff
@@ -5939,6 +5926,17 @@ int skill_castend_damage_id (struct block_list* src, struct block_list *bl, uint
 			skill_attack(BF_MAGIC,src,src,bl,skill_id,skill_lv,tick,flag);
 		break;
 
+	case LG_OVERBRAND: {
```

Comment:
from this video https://www.youtube.com/watch?v=0gI7Cwl8yvk

Overbrand has simple Circular AOE around caster.